zathura/shortcuts.c

```diff
@@ -1110,10 +1104,27 @@ bool sc_navigate_index(girara_session_t* session, girara_argument_t* argument, g
       gtk_tree_path_down(path);
     }
     break;
+  case EXPAND_RECURSIVE:
+    if (gtk_tree_view_expand_row(tree_view, path, TRUE)) {
+      gtk_tree_path_down(path);
+    }
+    break;
```

Comment:
The cases `EXPAND` and `EXPAND_RECURSIVE` could also be put together:

```suggestion
  case EXPAND:
  case EXPAND_RECURSIVE:
    if (gtk_tree_view_expand_row(tree_view, path, argument->n == EXPAND_RECURSIVE)) {
      gtk_tree_path_down(path);
    }
    break;
```

What do you think?